### PR TITLE
Add: Zsh

### LINKED ISSUE #8106: Request: Zsh
### Brand Name

Zsh

### Website

https://www.zsh.org

### Popularity Metric

The Similarweb rank is 2,206,442. See https://www.similarweb.com/website/zsh.org

### Official Resources for Icon and Color

- [Zsh Icon](https://raw.githubusercontent.com/Zsh-art/logo/main/svg/black_simple_icon.svg)
  https://github.com/Zsh-art/logo

- `#F15A24`

> from this comment
> https://github.com/simple-icons/simple-icons/issues/8106#issuecomment-1423247772
(Thanks👍)

### Additional Comments

Zsh is a default interactive shell of macOS from macOS Catalina.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by 5ouma**
zsh

**Issue:** closes #8106

**Similarweb rank:** 2,206,442

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

The Similarweb rank is quite low, but Zsh must be as popular as Bash (#2185).

Thank you @21st-centuryman, @jdorfman and @ferdnyc!

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15275,6 +15275,11 @@
             "hex": "CC2936",
             "source": "https://www.zotero.org/support/brand"
         },
+        {
+            "title": "Zsh",
+            "hex": "F15A24",
+            "source": "https://github.com/Zsh-art/logo/blob/7b54f2965cb2fc4d138db332e28283f0c6296850/svg/black_vertical_icon.svg"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:15281]:
```suggestion
            "source": "https://github.com/Zsh-art/logo/blob/253927e62bd2a640d29915ad7a794880b787eda8/svg/black_vertical_icon.svg"
```
This seems to be like the latest commit

File: icons/zsh.svg
Comment (PratyushJayachandran) [inline icons/zsh.svg:1]: There is a [slight difference](https://github.com/simple-icons/simple-icons/assets/35000807/c89f387c-7607-47cf-8cd6-2ec94d9e9921) when comparing against the [source.](https://github.com/Zsh-art/logo/blob/253927e62bd2a640d29915ad7a794880b787eda8/svg/black_vertical_icon.svg)

Can you please give it another go @5ouma

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Comment by jdorfman** [comment]
Looks great!

File: _data/simple-icons.json (hunk 1 above)
Comment (ferdnyc) [reply _data/simple-icons.json]: Oh... at the risk of delaying this further (sorry!), should this be using `https://github.com/Zsh-art/logo/blob/main/svg/black_simple_icon.svg` as the source, instead? (Latest commit permalink, `https://github.com/Zsh-art/logo/blob/253927e62bd2a640d29915ad7a794880b787eda8/svg/black_simple_icon.svg`.) That version was added to the repo specifically for compatibility with Simple Icons.

Comment (5ouma) [reply _data/simple-icons.json]: @ferdnyc
I think the difference has been made when you tried to compound all paths into one.
As my commit shows, the file that I retried to make and an original SVG file are completely the same.

Comment (ferdnyc) [reply _data/simple-icons.json]: @5ouma 

That's cool, whatever you prefer. If the newer version is the Most Right™ one, feel free to open a PR updating `black_simple_icon.svg` and I'm sure @jdorfman will merge it.

**Review by jdorfman — APPROVED** [review]

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for the contribution @5ouma 🎉

**Comment by 5ouma** [comment]
Thanks!!

**Pull request merged**